platform-services.md

```diff
@@ -0,0 +1,215 @@
+in its main activity.
+
+### App structure {#app-structure}
+
+The `hello_services` sample app has the following project file structure.
+
+ ![hello_services sample files](/images/hello_services_files.png)
+
+**Fig. 2.** Project file structure for the `hello_services` sample.
+
+To make platform-specific service calls, you may need to modify these key files and folders in the `hello_services` project:
+
+* [`android/app/src/main/java/`](https://github.com/flutter/flutter/tree/master/examples/hello_services/android/app/src/main/java):
+This folder contains the Java source files for the Android-specific part of your app. The file
+organization in this folder conforms to the Android app module format for Gradle. The main activity
+file
+([`ExampleActivity.java`](https://github.com/flutter/flutter/blob/master/examples/hello_services/android/app/src/main/java/com/example/flutter/ExampleActivity.java))
+contains the code that queries the Android system location API
+([`android.location.LocationManager`](https://developer.android.com/reference/android/location/LocationManager.html)),
+and returns the location data to the Flutter app.
+
+* [`android/app/src/main/AndroidManifest.xml`](https://github.com/flutter/flutter/blob/master/examples/hello_services/android/app/src/main/AndroidManifest.xml):
+The manifest file lists the Android activities that perform the services calls and any permissions
+that your app needs to access the services.
+
+* `android/local.properties`: Contains the configuration settings to build your Android app. See
+the `hello_services` [README](https://github.com/flutter/flutter/blob/master/examples/hello_services/README.md)
+to learn more about the properties you can configure.
+
+* [`ios/HelloServices/`](https://github.com/flutter/flutter/tree/master/examples/hello_services/ios/HelloServices):
+This folder contains the Objective-C source files for the iOS-specific part of your app. The
+[`AppDelegate.m`](https://github.com/flutter/flutter/blob/master/examples/hello_services/ios/HelloServices/AppDelegate.m)
+file handles message passing between iOS and the Flutter app, while the [`LocationProvider.m`](https://github.com/flutter/flutter/blob/master/examples/hello_services/ios/HelloServices/LocationProvider.m)
+file implements the code that queries the iOS system location API
+([`CLLocationManager`](https://developer.apple.com/library/ios/documentation/CoreLocation/Reference/CLLocationManager_Class/)).
+
+* `ios/Flutter/Generated.xcconfig`: Contains the configuration settings to build your iOS app. See
+the `hello_services` [README](https://github.com/flutter/flutter/blob/master/examples/hello_services/README.md)
+to learn more about the properties you can configure.
+
+* [`lib/main.dart`](https://github.com/flutter/flutter/blob/master/examples/hello_services/lib/main.dart):
+The main entry point for your Flutter app. This source file is written in Dart syntax. The main
```

Comment:
Again, "the Flutter part of your app".  Also, this section is confused about whether you're talking about `hello_services` or the developer's app.  Here you talk about "your app", but you're referencing hello_services specifically, which doesn't belong to the person reading this document.

"This source file is written in Dart syntax."  I'd strike this sentence.  If you really want to keep it, you should make it parallel to the other constructions and talk about the "Dart source files for the cross-platform part of your app".